website/tutorial/basic/02-getting-started.mdx

```diff
@@ -0,0 +1,325 @@
+---
+id: 02-getting-started
+title: Getting Started
+sidebar_label: Getting Started
+---
+
+In this chapter, you will learn about the GraphQL schema:
+
+1. How it performs as an API contract between the consumer and the provider,
+2. How you can use `graphql` library as a basic GraphQL execution mechanism
+3. What is a GraphQL operation and how you can use it.
+
+### Getting Started with GraphQL
+
+To get a better understanding of how GraphQL works, you can start by [reading this tutorial about GraphQL basics](https://www.howtographql.com/basics/0-introduction/).
+
+If you are already familiar with the basics of GraphQL, here's a quick overview:
+
+1. The GraphQL schema is where your GraphQL types are defined.
+1. GraphQL types are connected to each other using fields, and they form a graph.
+1. The `Query`, `Mutation` and `Subscription` types are special, since they act as entry point to the graph.
+1. The GraphQL schema acts as the data provider, and it offers a set of capabilities the consumer can use.
+1. To get data from a GraphQL schema, you need to write a GraphQL operation (often referred to as `query`) that selects the data and fields you need.
+
+In this section of the tutorial, you'll write a simple GraphQL schema, and you'll consume it directly, just for the learning process.
+
+Later, you'll replace the direct execution with a GraphQL server (based on HTTP protocol), and you'll add developer tools that will make it super simple to query and access.
+
+### Creating your first GraphQL schema
+
+There are many ways to create a GraphQL schema - in this tutorial, you are going to use schema-first approach, and build the schema with the `@graphql-tools/schema` library (take a look at the end of this chapter for more advanced/different solutions for creating your GraphQL schema).
+
+<Instruction>
+
+Start by installing `graphql` and `@graphql-tools/schema` library in your project, using the following command:
```

Comment:
we don't necessary need `@graphql-tools/schema` because Yoga can just take in `typeDefs` and `resolvers` and will just work 😎

Reply:
added this here: https://github.com/dotansimha/graphql-yoga/issues/916